person.rb

```diff
@@ -19,6 +23,10 @@ def correct_name
     @name
   end
 
+  def create_rental(date, book, person)
+    Rental.new(date, book, person)
+  end
```

Comment:
- [ ] The person itself should be sent as a parameter to create the new rental achieving the 'has-many' association. This can be achieved as you did in the book component. It's not necessary to set a person as a parameter just add the `self` to the Object Rental.

Reply:
Thank you for your feedback, I just made your suggestions.